rasa/model.py

```diff
@@ -218,7 +252,8 @@ def create_package_rasa(
         FINGERPRINT_CONFIG_NLU_KEY: _get_hash_of_config(
             config, include_keys=CONFIG_MANDATORY_KEYS_NLU
         ),
-        FINGERPRINT_DOMAIN_KEY: hash(domain),
+        FINGERPRINT_DOMAIN_WITHOUT_NLG_KEY: hash(domain_without_nlg),
```

Comment:
couldn't we do this?
```suggestion
        FINGERPRINT_DOMAIN_WITHOUT_NLG_KEY: get_dict_hash(domain_dict),
```

Reply:
We don't want the `templates` section to be included in the hash, right?